include/aspect/particle/interpolator/cell_average.h

```diff
@@ -52,6 +52,29 @@ namespace aspect
 
           // avoid -Woverloaded-virtual:
           using Interface<dim>::properties_at_points;
+
+          /**
+          * @copydoc Interface<dim>::declare_parameters()
+          **/
+          static
+          void
+          declare_parameters (ParameterHandler &prm);
+
+          /**
+           * @copydoc Interface<dim>::parse_parameters()
+           **/
+          virtual
+          void
+          parse_parameters (ParameterHandler &prm);
+
+        private:
+          /**
+           * Generally, particles need to be distributed throughout the
```

Comment:
This wording is not really clear to someone who is not familiar with how particles work. What about the following:
```
           * By default, every cell needs to contain particles to use this interpolator
           * plugin. If this parameter is set to true, cells are allowed to have no particles,
           * in which case the interpolator will return 0 for the cell's properties.
```